curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/step-059.md

```diff
@@ -22,7 +22,7 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('.question'));
 You should give the `.question` a `border` of `none`.
 
 ```js
-assert.equal(new __helpers.CSSHelp(document).getStyle('.question')?.border, 'none');
+assert.include(['none', 'medium none'], new __helpers.CSSHelp(document).getStyle('.question')?.border);
```

Comment:
```suggestion
assert.equal(new __helpers.CSSHelp(document).getStyle('.question')?.borderStyle, 'none');
```

Reply:
Interestingly, when using borderStyle, FF will now pass with either `border: none` or `border: 0` because it returns "none" for borderStyle in both cases. But Chrome will only pass with `border: none` because it returns "initial" for `border: 0`.

Reply:
For me that's okay, both essentially makes border not appear :man_shrugging: